File: assets/src/product.cpp
// Product sample

#include <iostream>

using namespace std;

class Product
{
protected:
    int price;
public:
    void SetPrice(int p)
    {
        this->price = p;
    }

    virtual int GetPrice()
    {
        return price;
    }

    virtual ~Product() {}
};

int main()
{
    Product* product = new Product();
    product->SetPrice(100);
    cout << "商品の価格は" << product->GetPrice() << "円" << endl;
    return 0;
}

File: assets/src/product.py
# Product sample
# coding: utf-8

class Product():
  price = 0
  def setPrice(self, p):
    self.price = p
  def getPrice(self):
    return self.price

product = Product()
product.setPrice(100)
print("商品の価格は" + str(product.getPrice()) + "円")

